<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="urlLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>631</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="keywordLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>631</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="urlLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="keywordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>Keyword</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>100</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>29</height>
     </size>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultsTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>780</width>
      <height>390</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>192</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>780</width>
      <height>390</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Results&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
